frontend/src/metabase/nav/containers/MainNavbar/MainNavbarContainer/MainNavbarView.tsx

```diff
@@ -104,6 +107,16 @@ function MainNavbarView({
     [canNavigateHome, onItemSelect],
   );
 
+  // Can upload CSVs if
+  // - properties.token_features.attached_dwh === true
+  // - retrieve collection using properties.uploads-settings.db_id
+  const hasAttachedDWHFeature = useSelector(
+    state => getSetting(state, "token-features").attached_dwh,
+  );
```

Comment:
```suggestion
  const hasAttachedDWHFeature = useHasTokenFeature("attached_dwh");
```

https://github.com/metabase/metabase/blob/93334133eabd167bbde56aa63db9b399a1a00184/frontend/src/metabase/common/hooks/use-has-token-feature/use-has-token-feature.ts#L5-L7